docs/Calendar.rst

```diff
@@ -887,399 +510,514 @@ feature can be applied to the date and time differently.
 
--  Click |image94| to get the RSS feed link which can be used in an RSS
-   reader application to directly view your calendar events and tasks.
+|AGENDA27|
 
+**The third way**
 
-.. _ManagingCategories:
+A new section is added to any event view you are invited to participate to in which you find if you are connected to your Google personal Calendar or not.
 
-====================
-Managing categories
-====================
+You need to click on the *Connect your personal calendar* button to choose it.
 
-Categories are used to classify events and tasks. For example, you could
-use a 'Meeting' category for all meetings with your colleagues or
-customers in your company. By default, PRODUCT provides 6 available
-categories in Calendar: All, Calls, Meeting, Holiday, Clients,
-Anniversary. You can add and view events/tasks in default categories. In
-addition, you also can edit, delete default categories or create new
-categories by yourself.
+|AGENDA28|
 
-To manage categories, click |image95|, then select Add Event Category
-from the drop-down menu.
+.. note:: If no personal calendar connector is displayed in the Personal Calendar drawer, an information message is displayed. 
+   |AGENDA29|.
+   you can take a look on :ref:`Manage Personal Calendar <ManagePersonalCalendar>` for 
+   more detail about enabling and displaying Personal Calendars.
+  
 
-The Event Categories form will appear.
+How can I connect my personal calendar ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-|image96|
+You can simply click on the *edit icon* in the Settings page, or Connect to my personal Calendar in the *Choose a date* stepper or *Event view*.
 
-.. _AddNewCategory:
+A drawer that contains the Calendars connectors is displayed.
 
-Adding a new category
-~~~~~~~~~~~~~~~~~~~~~~
+You have 2 choices: Google Calendar and Microsoft Outlook. 
+
+|AGENDA30|
+
+You are invited to click on connect in front of one of these choices. 
+
+If you are not connected to your personal calendar, you can click on You are invited to connect to your personal Calendar and the Connect to personal calendar drawer is opened so that you can connect to it.
+
+When you click on connect, a popup of your connected email in the browser is displayed, or you can connect to another email.
+
+|AGENDA31|
+
+.. note:: You are allowed to connect to only one personal calendar. 
+
+Once connected, you will have your personal connected Google email and Google avatar for Google account, and same for Outlook account.
+
+|AGENDA32|
+
+How can I view my personal events ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can view your personal events via either of these 2 views:
+
+**Choose a date stepper**:
+
+As long as your personal calendar is connected to the digital workplace, you are able to view your personal events in the Choose a date step when you create a new event so that you are aware about your events in the chosen space to create the event into and your Google personal events.
```

Comment:
```suggestion
As long as your personal calendar is connected to the digital workplace, you can view your personal events in the Choose a date step when you create a new event so that you are aware about your events in the chosen space to create the event into and your Google personal events.
```